src/query/plan/pretty_print.cpp

```diff
@@ -180,7 +180,7 @@ bool PlanPrinter::PreVisit(query::plan::Foreach &op) {
 }
 
 bool PlanPrinter::PreVisit(query::plan::Filter &op) {
-  WithPrintLn([](auto &out) { out << "* Filter"; });
+  WithPrintLn([&](auto &out) { out << "* " << op.ToString(); });
```

Comment:
```suggestion
  WithPrintLn([&op](auto &out) { out << "* " << op.ToString(); });
```
taking all as a reference is costly

Reply:
Done